lib/src/main/java/com/dynatrace/file/util/DynatraceFileBasedConfigurationProvider.java

```diff
@@ -98,7 +103,9 @@ private String tryGetToken(Properties props) {
       return null;
     }
     if (!newToken.equals(config.getMetricIngestToken())) {
-      logger.info("API Token refreshed.");
+      if (alreadyInitialized) {
```

Comment:
Do we really need this check? The first time the config has a value of `""`, so I guess that's what you want to avoid logging as it will *always* be different?